internal/runners/clean/run_win.go

```diff
@@ -22,20 +24,31 @@ func (u *Uninstall) runUninstall() error {
 		return err
 	}
 
-	err = removePaths(u.installPath, u.cfg.ConfigPath())
+	logFile, err := ioutil.TempFile("", "state-clean-uninstall")
+	if err != nil {
+		return locale.WrapError(err, "err_clean_logfile", "Could not create temporary log file")
+	}
+
+	err = removePaths(logFile.Name(), u.installPath, u.cfg.ConfigPath())
 	if err != nil {
 		return err
 	}
 
-	u.out.Print(locale.Tr("clean_message_windows", u.installPath, u.cfg.ConfigPath()))
+	u.out.Print(locale.Tr("clean_message_windows", u.installPath, u.cfg.ConfigPath(), logFile.Name()))
 	return nil
 }
 
-func removeConfig(configPath string) error {
-	return removePaths(configPath)
+func removeConfig(configPath string, out output.Outputer) error {
+	logFile, err := ioutil.TempFile("", "state-clean-config")
+	if err != nil {
+		return locale.WrapError(err, "err_clean_logfile", "Could not create temporary log file")
+	}
+
+	out.Print(locale.Tr("clean_config_message_windows", configPath, logFile.Name()))
```

Comment:
```suggestion
	out.Print(locale.Tr("clean_config_message_windows", logFile.Name()))
```

Reply:
Related suggestion further down (I don't think we need to be telling them the configPath, that should be in the log).